setup.py

```diff
@@ -71,7 +64,7 @@ def run_tests(self):
         "Topic :: Software Development :: Libraries"
     ],
     # Support for the basestring type is new in Cython 0.20.
-    setup_requires=["setuptools_cython", "Cython >= 0.20"],
+    install_requires=["Cython >= 0.20", "setuptools >= 18.0"],
```

Comment:
seems this does not work for setuptools. maybe we should add an assert to check setuptools version?